modules/public/static.go

```diff
@@ -18,8 +18,19 @@ import (
 	"time"
 
 	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/util"
 )
 
+// GlobalModTime provide a gloabl mod time for embedded asset files
+func GlobalModTime(filename string) time.Time {
+	t, err := util.GetExecutableModTime()
+	if err != nil {
+		log.Error("GetExecutableModTime failed:", err)
```

Comment:
If the first GetExecutableModTime fails, it will always fail, this this print a lot of helpless logs.

Maybe we should print a few logs at most